docs/compilers/osv.md

```diff
@@ -1,6 +1,9 @@
 # OSv Unikernels
 
 UniK uses OSv as a platform for compiling Java to unikernels.
+Additionally, UniK has a built-in support to directly run application packages provided in
+[MIKELANGELO Project](https://www.mikelangelo-project.eu)'s
+[format](https://github.com/mikelangelo-project/capstan/blob/develop/Documentation/ApplicationManagement.md#package-management).
```

Comment:
@miha-plesko I just opened the entire readme and noticed that there is a section on Java, which still relies on the version of the current compiler. I wonder if it would be better to have a separate readme or at least make sure that users understand that Java compiler is not currently using the MIKE project.